pkg/connector/connector_test.go

```diff
@@ -26,17 +27,34 @@ type connectorCase struct {
 
 var _ = Describe("EKS Connector", func() {
 	readResources := func() (connector.ManifestTemplate, error) {
-		connectorResources, err := ioutil.ReadFile("testdata/eks-connector.yaml")
+		readManifest := func(filename string) (connector.ManifestFile, error) {
+			data, err := ioutil.ReadFile(path.Join("testdata", filename))
+			if err != nil {
+				return connector.ManifestFile{}, nil
+			}
+			return connector.ManifestFile{
+				Data:     data,
+				Filename: filename,
+			}, nil
+		}
```

Comment:
nit: (dejavu) why is this a local func rather than a regular func :smile: ?

Reply:
`readManifest` doesn't have to be inside `readResources`, it was a mistake :smile: . I have moved it out.